x/auth/vesting/client/cli/tx.go

```diff
@@ -311,3 +312,33 @@ func NewMsgClawbackCmd() *cobra.Command {
 	flags.AddTxFlagsToCmd(cmd)
 	return cmd
 }
+
+// NewMsgReturnGrantsCmd returns a CLI command handler for creating a
+// MsgReturnGrantsCmd transaction.
+func NewMsgReturnGrantsCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "return-grants",
+		Short: "Transfer grants out of a vesting account.",
+		Long: `Must be authorized by the vesting account itself.
```

Comment:
Please strip leading whitespace in subsequent lines.

Reply:
Done - here and for the clawback command too. (Didn't change anything that's not an Agoric extension in order to avoid unneeded differences.)